src/services/Client.config.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Cosmo Tech.
```

Comment:
Can we discuss to find a better name than `Client.config.js` ? IMHO this is not a configuration file

Reply:
ClientApi.js is ok for you? I named it like the constant that we export in it

Reply:
Yup, sounds good to me :+1: